corehq/apps/app_manager/xform.py

```diff
@@ -976,6 +979,9 @@ def _get_label_ref(self, prompt):
         return None
 
     def resolve_path(self, path, path_context=""):
+        from corehq.apps.app_manager.models import SmartCaseUpdate
+        if type(path) is SmartCaseUpdate:  # TODO: this returns the right thing, but still need to update add_stuff_to_xform
```

Comment:
A nitpick, but if we expect this to become generally available/the new standard: since the "path" parameter in this method will no longer be a path but a SmartCaseUpdate, it might be better to name it something like `smart_case_update`, then instead of checking if it's a `smart_case_update`, assume that that it is one and instead check if it's just a path.

If it were just under a feature flag this would make sense because then we're adding a check "just in case", but I think the idea is it will become integrated with everything else and `SmartCaseUpdate` will be the new standard.